db/reqlog.c

```diff
@@ -3161,10 +3207,10 @@ static int dump_client_sql_data_single(void *ent, void *arg) {
 
 void dump_client_sql_data(struct reqlogger *logger, int do_snapshot) {
     struct dump_client_sql_options options = { .do_snap = do_snapshot, .logger = logger };
-    Pthread_rwlock_wrlock(&clientstats_lk);
+    acquire_clientstats_lock(0);
```

Comment:
should it be 1 here, for wrlock?

Reply:
`dump_client_sql_data_single` doesn't modify any data so I think this should be a read lock for efficiency